package.json

```diff
@@ -10,49 +10,10 @@
     "Alexandre Chau",
     "Julien Torrent"
   ],
-  "dependencies": {
-    "@emotion/react": "11.10.5",
-    "@emotion/styled": "11.10.5",
-    "@graasp/chatbox": "1.0.3",
-    "@graasp/query-client": "0.2.1",
-    "@graasp/sdk": "0.4.1",
-    "@graasp/ui": "0.11.2",
-    "@mui/icons-material": "5.11.0",
-    "@mui/lab": "5.0.0-alpha.119",
-    "@mui/material": "5.11.8",
-    "@sentry/react": "7.37.2",
-    "@sentry/tracing": "7.37.2",
-    "clsx": "1.2.1",
-    "http-status-codes": "2.2.0",
-    "i18next": "21.10.0",
-    "immutable": "4.2.4",
-    "js-cookie": "3.0.1",
-    "lodash.truncate": "4.4.2",
-    "prop-types": "15.8.1",
-    "qs": "6.11.0",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-ga4": "1.4.1",
-    "react-i18next": "11.18.6",
-    "react-intersection-observer": "9.4.0",
-    "react-query": "3.39.2",
-    "react-router": "6.2.1",
-    "react-router-dom": "6.2.1",
-    "react-scripts": "5.0.1",
-    "react-toastify": "9.0.3",
-    "uuid": "9.0.0"
-  },
   "scripts": {
-    "pre-commit": "yarn prettier:check && yarn lint",
-    "post-commit": "git status",
-    "start": "env-cmd -f ./.env.local react-scripts start",
-    "start:test": "env-cmd -f ./.env.test react-scripts start",
-    "start:prod": "env-cmd -f ./.env.production react-scripts start",
-    "start:ci": "react-scripts -r @cypress/instrument-cra start",
```

Comment:
What about coverage?

Reply:
Coverage with cypress in vite is handled differently than with CRA.
Actually, it is CRA that handled it differently (but this is another topic).

In CRA:
- use the "require" option of `react-scripts` to require packages execution before starting the app.
  - this was needed to instrument the app before it was run

In Vite:
- use `vite-plugin-istanbul` in vite config to instrument app code with istanbul. No Babel transforms needed.

What does not change:
- register the `@cypress/coverage/task` in the node events sections of the cypress config.

Hope it makes sense, feel free to follow up on any questions.

Reply:
You can check the coverage results in [the latest run](https://github.com/graasp/graasp-player/actions/runs/4355445034/jobs/7612073787) under the `coverage report` step.